src/overlords/defense/rangedDefense.ts

```diff
@@ -33,12 +33,15 @@ export class RangedDefenseOverlord extends CombatOverlord {
 	}
 
 	private handleDefender(hydralisk: CombatZerg): void {
-		if (this.room.hostiles.length > 0) {
-			hydralisk.autoCombat(this.room.name);
-		} else {
-			hydralisk.doMedicActions(this.room.name);
-		}
-	}
+	    if (this.room.hostiles.length > 0) {
+            hydralisk.autoCombat(this.room.name);
+        } else {
+            if (hydralisk.pos.getRangeTo(this.directive.pos) > 4) {
```

Comment:
`goTo` and `doMedicActions` both control movement, so running them both in the same tick is bad. also, these changes mean hydras won't heal up wounded creeps after a battle is over and will instead just clump in the middle of the room